app/models/user.rb

```diff
@@ -419,6 +419,7 @@ def seed_aspects
     self.aspects.create(:name => I18n.t('aspects.seed.friends'))
     self.aspects.create(:name => I18n.t('aspects.seed.work'))
     aq = self.aspects.create(:name => I18n.t('aspects.seed.acquaintances'))
+    aq.chat_enabled = true # enable chat aspect in acquaintances
```

Comment:
I feel like activate the chat for an aspect can be a good idea but should be discussed first so it should not be part of this PR.